test/Particle.spec.js

```diff
@@ -1227,11 +1221,10 @@ describe('ParticleAPI', () => {
 			it('allows invalidating tokens', () => {
 				return api.changeUserPassword({ auth: 'X', currentPassword: 'blabla', password: 'blabla2', invalidateTokens: true })
 					.then((results) => {
-						results.should.eql({
+						results.should.match({
```

Comment:
i'd rather be confident in _exactly_ what we're sending in and should's `.match()` behavior strikes me as a bit too clever ([docs](https://shouldjs.github.io/#assertion-match)). was `.eql()` causing issues?

Reply:
ah, i see - `.match()` is already being used in a bunch of places... 🤷‍♂️